test/index.spec.js

```diff
@@ -139,9 +139,8 @@ describe('FsDatastore', () => {
       }
     })
   })
-
-  // TODO: depends on sharding query fix
-  describe.skip('interface-datastore (sharding(fs))', () => {
+  
+  describe('interface-datastore (sharding(fs))', () => {
```

Comment:
RAD!

Reply:
Maybe not so rad, see #6, maybe the test belongs somewhere else.

Reply:
These tests do belong here, to make sure our implementation of `fs` passes the interface-datastore tests with sharding. Otherwise this repo can break things without ci noticing it

Reply:
okay, so I'll move the various dependencies into devDependencies.